envisage/tests/event_tracker.py

```diff
@@ -46,16 +47,18 @@ class EventTracker(HasTraits):
 
     #### Trait change handlers ################################################
 
-    def _subscriptions_changed(self, old, new):
+    @observe("subscriptions")
+    def _handle_changes_to_subscriptions(self, event):
         """ Static trait change handler. """
-
+        old, new = event.old, event.new
         for subscription in old:
             self._remove_subscription(subscription)
 
         for subscription in new:
             self._add_subscription(subscription)
 
-    def _subscriptions_items_changed(self, event):
+    @observe("subscriptions:items")
+    def _handle_changes_to_subscriptions_items(self, event):
```

Comment:
Maybe use this type of naming for the methods I commented on in `envisage/plugin_manager.py`

Reply:
i don't think that's a good idea. in fact, i'll try to rename these methods now - because the names are too generic and again, the methods denote what they are reacting to and now what they are going to do.

Reply:
renamed.